docs/configuration/settings.rst

```diff
@@ -507,6 +507,56 @@ dangerous to leave on by default, and must therefore be enabled explicitly.
 Then, issue a `POST` request to the `/__flush__` endpoint to flush all
 the data.
 
+
+.. _configuration-client-caching:
+
+Client caching
+==============
+
+In addition to :ref:`per-collection caching <collection-caching>`, it is possible
+to add cache control headers for every :app:`Kinto` object.
+The client (or cache server or proxy) will use them to cache the collection
+records for a certain amount of time, in seconds.
+
+The setting can be set for any kind of object (``bucket``, ``collection``, ``record``),
+and concerns read-only requests (``GET /buckets``, ``GET /buckets/{}/collections``,
+``GET /buckets/{}/collections/{}/records``).
+
+.. code-block:: ini
+
+    # cliquet.bucket_cache_expires_seconds = 3600
+    # cliquet.collection_cache_expires_seconds = 3600
+    cliquet.record_cache_expires_seconds = 3600
+
+
+If set to ``0`` then the resource becomes uncacheable (``no-cache``).
+
+
+Nginx
```

Comment:
This probably shouldn't go in this section but in the "recipes" for deployment instead. We could link there from here.